proto/vtctldata.proto

```diff
@@ -811,6 +813,60 @@ message InitShardPrimaryResponse {
   repeated logutil.Event events = 1;
 }
 
+message MoveTablesCreateRequest {
+  // The necessary info gets passed on to each primary tablet involved
+  // in the workflow via the CreateVRWorkflow tabletmanager RPC.
+  string workflow = 1;
+  string source_keyspace = 2;
+  string target_keyspace = 3;
+  repeated string cells = 4;
+  repeated string tablet_types = 5;
+  repeated string source_shards = 6;
+  bool all_tables = 7;
+  repeated string include_tables = 8;
+  repeated string exclude_tables = 9;
```

Comment:
why do we need `all_tables`? how is that different from an empty/unset `include_tables` and empty/unset `exclude_tables`?

Reply:
This is passing on the client command options to VtctldServer. It's only on the VtctldServer side that we know what All Tables means. From there, we don't use all tables (in the CreateVRWorkflow request).

Reply:
i think i'm still confused why we need this then

Reply:
We could treat an empty list of include and exclude tables as being ALL. But then you lose the current ability to e.g. say `--all --exclude-tables=foo` if e.g. you have 100 tables and want to move all of them but `foo`.